Update stack.yaml

fisx opened the pull request:
I'm not sure about the .ghci file as it is after this change, but it works for me, and I'm pretty sure it doesn't work for anybody before this.  I'm also ok with removing it entirely.

I also put the generated stack.yaml.lock under version control, to make builds accross time and space more deterministic.  I'm tempted to say I'm also ok with removing stack support...  :)

phadej commented:
We removed `.ghci`, so there's conflict. Not sure if `stack.yaml` is still up to date in this PR?

fisx commented:
> We removed `.ghci`, so there's conflict. Not sure if `stack.yaml` is still up to date in this PR?

It was, sort of.  I bumped it again and removed the .ghci.

thanks!

swamp-agr commented:
Closing it in favor of #228.
